packages/create-platformatic-db/src/README.md

```diff
@@ -0,0 +1,23 @@
+# Platformatic DB API
+
+This is a template application for [Platformatic DB](https://oss.platformatic.dev).
+
+## Requirements
+
+Platformatic supports macOS, Linux and Windows ([WSL](https://docs.microsoft.com/windows/wsl/) recommended).
+You'll need to [Node.js](https://nodejs.org/) >= v16.17.0 or >= v18.8.0
+
+## Usage
```

Comment:
We should include details on creating and running migrations here too, including links to docs. This is especially important if the user chose not to generate or apply migrations during the app generation process.